### DISCUSSION TITLE
Pregenerated Data folder

### DISCUSSION THREAD

**Discussion by kamilkrukowski** [kamilkrukowski/EDGAR-DOC-PARSER#58] [Announcements]
For those with proper permissions it is [here.](https://drive.google.com/file/d/1Hwqfhoi-5OO_ycczKd_BaJTUzILpE4Ww/view?usp=share_link)

There is ```zip``` password -- it is ```tufts```

**Comment by kamilkrukowski** [comment]
@kelseyyew @NC-Zhao @yyitingli Can each of you let me know when you successfully download and use this folder on your local machines?

**Comment by yyitingli** [comment]
@kamilkrukowski 
It asks me for a password to decompress the zip file.
<img width="348" alt="Screen Shot 2022-11-15 at 2 59 47 PM" src="https://user-images.githubusercontent.com/66972539/202014035-31b0b0ce-89e8-4774-9446-9922c085abb8.png">

**Reply by kamilkrukowski** [reply]
I will fix this after a meeting (by 5pm today) and regenerate the hyperlink then ping you once it is fixed.

**Reply by yyitingli** [reply]
Thanks!

**Reply by kamilkrukowski** [reply]
@yyitingli fixed

**Reply by kamilkrukowski** [reply]
Does it work @yyitingli ?

**Reply by yyitingli** [reply]
It works, thank you!
